marketplace/src/app/layout.tsx

```diff
@@ -24,6 +35,16 @@ export default function RootLayout({
 }) {
   const queryClient = new QueryClient();
 
+  mbjs.config({
```

Comment:
what's this doing?

Reply:
it is setting the network on mbjs packages